.vsts.ci.yml

```diff
@@ -22,7 +22,7 @@ parameters:
 - name: rhel6_x64
   type: boolean
   displayName: RHEL 6 (x64)
```

Comment:
Should this be updated to RHEL 7?

Reply:
Yes, we have other PR for that - https://github.com/microsoft/azure-pipelines-agent/pull/3771